util/brew-update

```diff
@@ -1,13 +1,20 @@
 #!/usr/bin/env ys-0
 
-releases-url =: 'https://github.com/yaml/yamlscript/releases'
-brew-url =: 'git@github.com:yaml/homebrew-yamlscript'
+releases-url =: "https://github.com/yaml/yamlscript/releases"
```

Comment:
In YAML, YS and shell code I don't use double quotes without a need for them. iow, always prefer single quotes.
There are many good reasons to use double quotes  but none here.